src/Compilers/CSharp/Portable/Parser/LanguageParser.cs

```diff
@@ -5346,6 +5343,13 @@ private bool IsTrueIdentifier()
             return false;
         }
 
+        private bool IsTrueIdentifier(SyntaxToken token)
```

Comment:
>IsTrueIdentifier [](start = 21, length = 16)

Should the method above delegate its work to this method? Otherwise, the methods have the same name, but appear to use different criteria.